app/assets/javascripts/discourse/tests/acceptance/tags-test.js

```diff
@@ -429,6 +432,28 @@ acceptance("Tag info", function (needs) {
     );
   });
 
+  test("edit tag is showing input for name and description", async function (assert) {
+    updateCurrentUser({ moderator: false, admin: true });
+
+    await visit("/tag/happy-monkey");
+    assert.strictEqual(count("#show-tag-info"), 1);
+
+    await click("#show-tag-info");
+    assert.ok(exists(".tag-info .tag-name"), "show tag");
+
+    await click("#edit-tag");
+    assert.equal(
```

Comment:
There will be a eslint rule enabled for than soon-ish

```suggestion
    assert.strictEqual(
```